package proto_ff;

// 命令字
enum Proto_CS_CMD {
	E_CS_CMD_NONE		                                = 0;
	E_CS_ERROR			                                = 1; //统一错误包回包


	//////////////1100-1149Login/////////////////////////////
	//1150--1169邮件//////////////////////////////////////////////////////////////////////////////////
	//1170-1189 银行//////////////////////////////////////////////////////////////////////////////////

	////////////////////////////////1200-1220rank///////////////////////////////////////////////

	NF_CS_MSG_UserStatusReq    = 10017;
	NF_SC_MSG_UserStatusRsp    = 10018;
};

message Proto_CSErrorRsp
{
	optional uint64 user_id = 1;
	optional uint32 req_msg_id  = 2;	//请求的命令号
	optional int32 error = 3; 	//错误码
}


